delivery_package_gs1_barcode_report/data/ir_sequence.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="stock.seq_quant_package" model="ir.sequence">
+        <field name="use_python_code">True</field>
+        <field
+            name="python_code"
+        >"387193340307" + number_padded + str(get_barcode_check_digit("387193340307" + number_padded + "x"))</field>
```

Comment:
Here, you are specifying a code that's specific to (probably) the person that you made it for. But, people need to be able to choose it for themselves.

I suggest to split up the module in two parts.

1. A module `sequence_sscc` that contains this part and of which the main use is to help people to automatically assign SSCC codes to a package, from a certain range that they have bought.
2. The current module, which includes the package number in the barcodes IF it is a valid SSCC code, and otherwise just omit the barcode.

--> In this way, you will have the both separated and people who for example input their SSCC code manually, can use the module without this functionality of auto generating. Also, you will lose the dependency on `sequence_python` and the module will become easier to review and to use.